public/app/plugins/panel/alertlist/UnifiedAlertList.tsx

```diff
@@ -456,3 +450,16 @@ export const getStyles = (theme: GrafanaTheme2) => ({
     display: none;
   `,
 });
+
+export function UnifiedAlertListPanel(props: PanelProps<UnifiedAlertListOptions>) {
+  if (
+    !contextSrv.hasPermission(AccessControlAction.AlertingRuleRead) &&
```

Comment:
We could use `useAlertingAbility(AlertingAction.ViewAlertRule)` and `useAlertingAbility(AlertingAction.ViewExternalAlertRule);` here instead :)